osu.Game/Online/API/Requests/Responses/APIUser.cs

```diff
@@ -34,8 +34,24 @@ public class APIUser : IEquatable<APIUser>, IUser
         [JsonProperty(@"previous_usernames")]
         public string[] PreviousUsernames;
 
+        private Country? country;
+
+        public Country Country
+        {
+            get => country ??= (Enum.TryParse(userCountry.Code, out Country result) ? result : default);
+            set => country = value;
+        }
+
+#pragma warning disable 649
         [JsonProperty(@"country")]
-        public Country Country;
+        private UserCountry userCountry;
+
+        private class UserCountry
+        {
+            [JsonProperty(@"code")]
+            public string Code;
+        }
+#pragma warning restore 649
```

Comment:
Would eventually hope to get rid of this and have osu-web provide the country code in a `country_code` property on the user object directly.